custom_components/tuya_local/devices/forceClima_Soundless.yaml

```diff
@@ -0,0 +1,86 @@
+name: ForceClima Soundless
```

Comment:
This top level name should be generic for the type of device, as Tuya devices are often sold as whitelabeled, so have different brands in different markets.

Reply:
That should be ok

Reply:
Fix